City-of-Helsinki/parking-permits-ui
# Pv 779 extend fixed permits

## Description

Allow customer to extend a permit.

## Context




[PV-779](https://helsinkisolutionoffice.atlassian.net/browse/PV-779)

## How Has This Been Tested?

Manual testing 

## Manual Testing Instructions for Reviewers

Requires PV-779 branch of backend https://github.com/City-of-Helsinki/parking-permits

1. Create a fixed term permit in Webshop for 1 month
2. Adjust the end time of permit in Django admin so that it ends within 14 days of current time
3. Click "Jatka pysäköintitunnuksen voimassaoloa" button on home page (this should appear if period time is correct, otherwise hidden).
4. Select the number of months. Pricing should appear, grouped by product with total amount.
5. Click "Jatka" to view order in Talpa. The extended request should also be in Django admin.


## Screenshots

![Screenshot from 2024-02-12 15-55-57](https://github.com/City-of-Helsinki/parking-permits-ui/assets/131681805/170ce5e9-8635-434b-b1cb-b8a45e05bcc7)


![Screenshot from 2024-02-12 16-00-32](https://github.com/City-of-Helsinki/parking-permits-ui/assets/131681805/be101b11-a583-4482-bbc1-bbf3115fd7f2)


[PV-779]: https://helsinkisolutionoffice.atlassian.net/browse/PV-779?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
